firedrake/mg/mesh.py

```diff
@@ -173,10 +173,10 @@ def MeshHierarchy(mesh, refinement_levels,
                          refinements_per_level, nested=True)
 
 
-def ExtrudedMeshHierarchy(base_hierarchy, layers, kernel=None, layer_height=None,
-                          extrusion_type='uniform', gdim=None,
+def ExtrudedMeshHierarchy(base_hierarchy, height, base_layer=-1, refinement_ratio=2, layers=None,
+                          kernel=None, extrusion_type='uniform', gdim=None,
```

Comment:
Please add documentation for the new arguments to the docstring. I realise that not all the existing arguments are documented, but that's not really an excuse for making things worse.

Reply:
Agreed, thanks for pointing it out. I have added docstrings.